### ISSUE TITLE
Edit Goal - My current goals

### ISSUE THREAD

**Issue by steffipravasi** [bcgov/performance#1255]

**Comment by jizhaogit** [comment]
Items need to talk if need to separate to new tickets:
1. ckeditor accessibility for comments and reply
2. tags multi-select
3. form's error message

**Comment by steffipravasi** [comment]
@jizhaogit Assigning ticket back to you for some fixes

**Comment by jizhaogit** [comment]
issue #3, #4, and #9 are addressed. thanks!

**Comment by steffipravasi** [comment]
@jizhaogit Can you please confirm if the fixes are pushed to Test

**Comment by steffipravasi** [comment]
@jizhaogit The fix are not reflected on test for issue 3,4,9

**Comment by jizhaogit** [comment]
checked with @steffipravasi,  and got details:
1) #3 and #4 has arial-label set in the html but screen reader does not read them
2) there is one issue detected in the date validation is if a user picks start date and end date, and then clears the start date, the form will still be submitted.

**Comment by jizhaogit** [comment]
those detected issues should be addressed

**Comment by steffipravasi** [comment]
Issues 3,4 & 9 have been addressed and verified
